### PR TITLE
create simple event

### PR DESCRIPTION
**Pull request by VoTaiTri**

### CODE REVIEW

File: app/controllers/schedules_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  def create
+    @schedule = Schedule.new schedule_params
+    @schedule.user = current_user
+
+    if @schedule.save
+      flash[:success] = "Create new event successfuly!"
+      redirect_to root_path
+    else
+      flash.now[:danger] = "False"
```
Comment (dieunb) [inline app/controllers/schedules_controller.rb]: Should write more clear message

**Comment by khuongnt-0294** [comment]
ok rui

File: config/routes.rb [hunk 2]
```diff
@@ -6,4 +6,6 @@
 
   namespace :admin do
   end
-end
+
+  resource :schedules
```
Comment (dieunb) [inline config/routes.rb]: only, except?

File: app/controllers/schedules_controller.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      redirect_to root_path
+    else
+      flash.now[:danger] = "Create new event false!"
+      render :new
+    end
+  end
+
+  private
+  def schedule_params
+    params.require(:schedule).permit :event, :date, :start_time, :finish_time,
```
Comment (dieunb) [inline app/controllers/schedules_controller.rb]: state???

**Pull request merged**